ee/codegen/src/__test__/nodes/inline-prompt-node.test.ts

```diff
@@ -422,4 +422,36 @@ describe("InlinePromptNode", () => {
       expect(await writer.toStringFormatted()).toMatchSnapshot();
     });
   });
+
+  describe("with functions attribute", () => {
+    it("should generate functions field with WorkflowValueDescriptor for upstream node output", async () => {
+      const functionsAttribute: NodeAttributeType = {
+        id: uuidv4(),
+        name: "functions",
+        value: {
+          type: "NODE_OUTPUT",
+          nodeId: "upstream-node-id",
+          nodeOutputId: "functions-output-id",
```

Comment:
You'll need to define the upstream node in this test too. See some of our other codegen examples with upstream nodes